### PR TITLE
fix casadi code generation (issue 162)

### PR DESCRIPTION
**Pull request by schoelst**
only very minor changes

### CODE REVIEW

File: swig/acados.i [hunk 1]
```diff
@@ -177,6 +177,7 @@ static std::string generate_vde_function(casadi::Function& model) {
     casadi::Function vde = casadi::Function(full_name, input, output);
     casadi::Dict opts;
     opts["with_header"] = casadi::GenericType(true);
+    opts["with_export"] = casadi::GenericType(false);
```
Comment (jgillis) [inline swig/acados.i]: Out of curiosity; why do you write explicit type-conversion to ` casadi::GenericType`?

Comment (roversch) [reply swig/acados.i]: Good question, I was wondering the same. `true` should work as well, no?

Comment (schoelst) [reply swig/acados.i]: I remember getting an error at some point and just sticked to it.

Comment (roversch) [reply swig/acados.i]: I merged it and fixed it locally.

Just `true` instead of `GenericType(true)` works for me.

**Pull request merged**